kola/tests/misc/raid.go

```diff
@@ -23,30 +23,63 @@ import (
 	"github.com/coreos/mantle/kola/register"
 	"github.com/coreos/mantle/platform"
 	"github.com/coreos/mantle/platform/conf"
+	"github.com/coreos/mantle/platform/machine/qemu"
 )
 
-func init() {
-	register.Register(&register.Test{
-		Run:         RootOnRaid,
-		ClusterSize: 1,
-		MinVersion:  semver.Version{Major: 1520},
-		Name:        "coreos.disk.raid.root",
-		UserData: conf.ContainerLinuxConfig(`storage:
+var (
+	raidRootUserData = conf.ContainerLinuxConfig(`storage:
+  disks:
+    - device: /dev/vdb
+      wipe_table: true
+      partitions:
+       - label: root1
+         number: 1
+         size: 256MiB
+         start: 0
+         type_guid: be9067b9-ea49-4f15-b4f6-f36f8c9e1818
+       - label: root2
+         number: 2
+         size: 256MiB
+         start: 0
+         type_guid: be9067b9-ea49-4f15-b4f6-f36f8c9e1818
   raid:
-    - name: "test-hasroot"
+    - name: "rootarray"
       level: "raid1"
       devices:
-        - "/dev/disk/by-partlabel/ROOT"
-        - "/dev/disk/by-partlabel/USR-B"
+        - "/dev/vdb1"
+        - "/dev/vdb2"
   filesystems:
     - name: "ROOT"
       mount:
-        device: "/dev/md/test-hasroot"
+        device: "/dev/md/rootarray"
+        format: "ext4"
+        create:
+          options:
+            - "-L"
+            - "ROOT"
+    - name: "NOT_ROOT"
+      mount:
+        device: "/dev/vda9"
         format: "ext4"
         create:
           options:
             - "-L"
-            - "ROOT"`),
+            - "wasteland"
+          force: true`)
+)
+
+func init() {
+	register.Register(&register.Test{
+		// This test needs additional disks which is only supported on qemu since Ignition
+		// does not support creating partitions on disks without wiping the partition table
```

Comment:
It does.  It just doesn't support deleting the ones that are currently filling up the disk.